requirements-dev.txt

```diff
@@ -1,4 +1,7 @@
--r ./requirements.txt
```

Comment:
We need this. Please do not remove the installation of the themes dependencies.

Reply:
This is related to the line/comment below (`-e .`) That line will install the package (and consequently its dependencies).

Reply:
Ah, yea, that makes sense.